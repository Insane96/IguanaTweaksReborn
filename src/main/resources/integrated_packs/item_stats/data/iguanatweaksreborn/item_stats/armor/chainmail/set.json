{
    "item": "#iguanatweaksreborn:equipment/armor/chainmail",
    "enchantability": 13,
    "armor_toughness": 0,
    "knockback_resistance": 0.04
}