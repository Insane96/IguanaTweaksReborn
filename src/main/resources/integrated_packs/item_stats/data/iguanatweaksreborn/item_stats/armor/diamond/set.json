{
    "item": "#iguanatweaksreborn:equipment/armor/diamond",
    "enchantability": 7,
    "armor_toughness": 0,
    "movement_speed_penalty": 0.08
}