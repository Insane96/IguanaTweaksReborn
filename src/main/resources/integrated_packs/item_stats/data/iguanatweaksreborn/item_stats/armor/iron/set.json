{
    "item": "#iguanatweaksreborn:equipment/armor/iron",
    "enchantability": 11,
    "armor_toughness": 0,
    "knockback_resistance": 0.06,
    "movement_speed_penalty": 0.05
}