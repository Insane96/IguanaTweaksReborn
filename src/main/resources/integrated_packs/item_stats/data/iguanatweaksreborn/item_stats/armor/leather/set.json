{
    "item": "#iguanatweaksreborn:equipment/armor/leather",
    "enchantability": 18,
    "armor_toughness": 4
}