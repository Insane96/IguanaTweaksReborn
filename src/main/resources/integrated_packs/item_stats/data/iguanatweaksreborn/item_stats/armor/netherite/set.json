{
    "item": "#iguanatweaksreborn:equipment/armor/netherite",
    "enchantability": 5,
    "armor_toughness": 0,
    "knockback_resistance": 0,
    "movement_speed_penalty": 0.12
}