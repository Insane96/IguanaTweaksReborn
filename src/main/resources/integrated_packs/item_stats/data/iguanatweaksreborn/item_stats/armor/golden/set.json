{
    "item": "#iguanatweaksreborn:equipment/armor/golden",
    "enchantability": 18,
    "armor_toughness": 0
}